pages/performances/[id].js

```diff
@@ -22,7 +22,7 @@ import { formatPerformance } from '@utils/performances'; // Format performance u
 export default function PerformanceDetails({ session }) {
   const router = useRouter();
   const { id } = router.query;
-  const { event: eventId } = Navigation.useContainer();
+  const [{ id: eventId }] = Event.useContainer();
```

Comment:
Some places have event and some places have id: eventId.  Is this intentional?

Reply:
Yes